src/Entity/MailQueueEntry.php

```diff
@@ -9,5 +9,5 @@
  * @ORM\Entity
  */
 class MailQueueEntry extends AbstractMailQueueEntry {
-	use \Kdyby\Doctrine\Entities\Attributes\Identifier;
-}
\ No newline at end of file
+	use  \ADT\MailQueue\Traits\Identifier
```

Comment:
is the trait even needed? It's only used here.

Reply:
It's not necessary, but it's a substitute for the identifier trait of kdyby.